node-runner-cli/commands/systemdcommand.py

```diff
@@ -46,11 +46,10 @@ def systemdcommand(systemdcommand_args=None, parent=systemd_parser):
              required=True,
              help="""Quick config mode with assumed defaults. It supports two quick modes and a detailed config mode.
               \n\nCORE: Use this value to setup CORE using defaults.
-              \n\nDETAILED: Default value if not provided. This mode takes your through series of questions.
               \n\nGATEWAY: This mode adds questions regarding the Network Gateway API and enables it for installation
               \n\nMIGRATION: This mode adds questions regarding the migration from an Olympia End-State node to a Babylon node
               """,
-             choices=["CORE", "DETAILED", "MIGRATION", "GATEWAY"], action="store"),
```

Comment:
We don't have any custom code for DETAILED mode. Hence removing it as that will fail now